docs/modules.md

```diff
@@ -4,4 +4,86 @@ title: "Player modules"
 
 # Modules
 
-TO BE DONE
+Playable is a highly expandable product. By default, it is shipped with certain included features. Anyway, basic functionality can be extended using a built-in modular system.
+
+## Module connection
+```javascript
+import Playabale from 'playable';
+import Module from 'playable/dist/modules/my-module';
+
+// include module before creating instance of player 
+Playable.registerModule('myModule', Module);
+// ...
+const player = Playable.create(config, theme);
+// ...
+```
+Like in the case with  [playback adapters](/adapters) there are two ways to add modules: via bundle or using `import`.
+<br /> // TODO: example with bundle;
+
+## API
+Module can extend players API, add new methods, events, etc.
+
+# List of available modules
+ - [chromecast](#chromecast)
+
+
+## Chromecast
+Chomecast module is an extension that connects playable to cast video files to google chromecast, SmartTV or compatible devices using google chrome browser API.
+If there is a device suitable device in your network and browser is able to start. Уou should see a corresponding control in a bottom panel of the player.
+### Add module
+```javascript
+import Playabale from 'playable';
+import ChromecastManager from 'playable/dist/modules/chromecast-manager';
+import ChromecastButton from 'playable/dist/statics/modules/ui/controls/chromecast';
```

Comment:
I don't see a reason to have 2 modules. Don't understand from the doc the responsibilities of them.
Looks like paths are misplaced. `dist/modules`/`dist/statics/modules`